### PR TITLE
[feature] Logout time for AutoLog.

### PR DESCRIPTION
**Pull request by scorbett123**
**Describe the pull**
Adds a logout time to AutoLog.

**Describe how this pull is helpful**
It gives the user additional context about when the logout happens if they are afk or using baritone.

**Additional context**
Suggested by #1602. 

If anyone wants any changes to the colour of the text or the position then please state them.

### CODE REVIEW

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoLog.kt [hunk 1]
```diff
@@ -96,7 +98,7 @@ object AutoLog : Module() {
         mc.connection?.networkManager?.closeChannel(TextComponentString(""))
         mc.loadWorld(null as WorldClient?)
 
-        mc.displayGuiScreen(KamiGuiDisconnected(reasonText, screen, disable.value == DisableMode.ALWAYS || (disable.value == DisableMode.NOT_PLAYER && reason != PLAYER)))
+        mc.displayGuiScreen(KamiGuiDisconnected(reasonText, screen, disable.value == DisableMode.ALWAYS || (disable.value == DisableMode.NOT_PLAYER && reason != PLAYER), LocalTime.now()))
```
Comment (l1ving) [inline src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoLog.kt:101]: which format is LocalTime.now(), ISO8601?

**Review by l1ving — APPROVED** [review]
Looks good if the format is fine, pending testing

File: src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoLog.kt (hunk 1 above)
Comment (scorbett123) [reply src/main/java/me/zeroeightsix/kami/module/modules/combat/AutoLog.kt:101]: yes.

**Review by l1ving — APPROVED** [review]

**Comment by l1ving** [comment]
Thank you :3

**Pull request merged**